README.md

```diff
@@ -41,27 +41,21 @@ result = randdev(n)
 @test length(result) == n
 ```
 
-How could we create a test that shows the output of the function is reversed? Mocking.jl
-provides the `@mock` macro which allows package developers to temporarily overload a
-specific calls in their package. In this example we will apply `@mock` to the `open` call
-in `randdev`:
+How could we create a test that shows the output of the function is reversed?
+ Mocking.jl provides a mechanism which allows package developers to temporarily overload a
+specific calls in their package. In this example we will mock the `open` call
+in `randdev`.
+No changes are required at the call site.
 
-```julia
-using Mocking
-
-function randdev(n::Integer)
-    @mock open("/dev/urandom") do fp
-        reverse(read(fp, n))
-    end
-end
-```
```

Comment:
The original function should be shown.

Reply:
The original function is already shown further up in the readme